src/service/video-manager-impl.js

```diff
@@ -171,9 +228,25 @@ class VideoEntry {
     /** @private {boolean} */
     this.hasAutoplay_ = element.hasAttribute(VideoAttributes.AUTOPLAY);
 
+    /** @private {boolean} */
+    this.userInteracted_ = !this.hasAutoplay_;
+
     listenOncePromise(element, VideoEvents.LOAD)
       .then(() => this.videoLoaded_());
 
+
+    const updatePlayStatus = playing => {
```

Comment:
please add two new private instance methods `videoPlayed_` and `videoPaused_` (soon will will be doing more in these methods than just setting playing state - e.g. trigger analytics)